config/locales/en.yml

```diff
@@ -1649,13 +1651,10 @@ en:
         client_secret: Client Secret
         automatically_approve_accounts: 'Always approve accounts that sign up through this SSO Integration'
         published_admin_portal: Display this SSO Integration on the sign-in page of the admin portal.
-      authentication_provider/github:
-        <<: *authentication_provider
-      authentication_provider/auth0:
-        <<: *authentication_provider
+        skip_ssl_certificate_verification: Skip SSL certificate verification
```

Comment:
The `skip_ssl_certificate_verification: Do not verify SSL certificate` translation wasn't working because of the reasons described in https://github.com/3scale/porta/pull/3984/files#r1925326970

I guess it used to work (as this PR suggests https://github.com/3scale/system/pull/8410), but got broken at some point.

And this was shown as `Skip ssl certificate verification`.

I decided to keep it like this, just with the correct capitalization, because this field was actually referenced this way in some other places.